state/container.go

```diff
@@ -140,6 +160,74 @@ func (c *container) String() string {
 		c.id, int(c.initPid), c.uidFirst, c.gidFirst)
 }
 
+func (c *container) IsImmutableMountID(id int) bool {
+	c.intLock.RLock()
+	defer c.intLock.RUnlock()
+
+	if info := c.mountInfoParser.LookupByMountID(id); info != nil {
```

Comment:
Suggestion: add comment:
"All mounts initially present in the container are considered immutable."